app/views/DefaultBranchListPage.scala.html

```diff
@@ -44,32 +50,51 @@ <h1>Default Branch</h1>
 <div id="default-branches">
     <form action="/defaultbranch" method="get">
         <div class="form-group row">
-            <div class="col-xs-1 padding-reset-right">
-                <label for="search">Name:</label>
-            </div>
             <div class="col col-xs-3">
-                <input class="search form-control" id="search" type="text" name="name" value='@form("name").value' autofocus>
-            </div>
-            <div class="col-xs-1 padding-reset-right">
-                <label for="teamNames">Team:</label>
+                <label for="search">Name:</label>
+                <div>
+                    <input class="form-control" id="search" type="text" name="name" value='@form("name").value'>
```

Comment:
there are form functions for displaying input fields (for text checkbox etc) which should make it simpler to sync to the form value.